eZ/Bundle/EzPublishCoreBundle/Tests/DependencyInjection/Compiler/ComplexSettingsPassTest.php

```diff
@@ -34,7 +34,7 @@ public function testProcess()
 
         $expressionString = 'service("ezpublish.config.complex_setting_value.resolver").resolveSetting("/mnt/nfs/$var_dir$/$storage_dir$", "var_dir", service("ezpublish.config.resolver").getParameter("var_dir", null, null), "storage_dir", service("ezpublish.config.resolver").getParameter("storage_dir", null, null))';
         $arguments = $definition->getArguments();
-        self::assertSame(1, count($arguments));
+        self::assertCount(1, $arguments);
```

Comment:
Side, but there is a CS fixer rule for this one:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/commit/6a71885257a1d4389e0a9e94620f9644767cf1e2

 Maybe get @alongosz to add it across as a followup to the current CS fix update :)

Reply:
yes, this was done by CS fixer, and new rules will be added

Reply:
yep, but one step at the time to avoid huge diffs, added this to my list of rules to be enabled later on ;)

Reply:
actually... since I've just merged #2647 into all maintained branches, there's nothing stopping us from configuring these rules right now. I needed to do a rebase anyway. Added rules via 7bfb329.